Makefile

```diff
@@ -28,21 +26,21 @@ SHELL := $(shell command -v bash 2>/dev/null)
 
 .DEFAULT: runc
 
-runc: $(SOURCES) | $(RUNC_LINK)
+runc: $(SOURCES)
 	go build -i -ldflags "-X main.gitCommit=${COMMIT} -X main.version=${VERSION}" -tags "$(BUILDTAGS)" -o runc .
 
 all: runc recvtty
 
 recvtty: contrib/cmd/recvtty/recvtty
 
-contrib/cmd/recvtty/recvtty: $(SOURCES) | $(RUNC_LINK)
+contrib/cmd/recvtty/recvtty: $(SOURCES)
 	go build -i -ldflags "-X main.gitCommit=${COMMIT} -X main.version=${VERSION}" -tags "$(BUILDTAGS)" -o contrib/cmd/recvtty/recvtty ./contrib/cmd/recvtty
 
-static: $(SOURCES) | $(RUNC_LINK)
+static: $(SOURCES) .GOPATH/.ok
```

Comment:
removed. Story is that at work we have a makefile template exactly to do these kind of symlink trick ( if someone wants symlink ) https://github.com/cloudflare/hellogopher. And so i tried to integrate it here in some form. But then i feel like symlink isnt necessary here, so remove it.